### ISSUE TITLE
BUG: Blank PDF viewer

### ISSUE THREAD

**Issue by JessicaMarine** [ustaxcourt/ef-cms#7702]
**Describe the Bug**
When an external user tries to eFile a document larger than 2MB, the PDF viewer modal on the "Review Your Filing" screen is blank.

**Business Impact/Reason for Severity**

**In which environment did you see this bug?**
Court's prod; able to reproduce in Court's mig

**Who were you logged in as?**
IRS attorney; similar issue also reported by a petitioner's counsel

**What were you doing when you discovered this bug? (Using the application, demoing, smoke tests, testing other functionality, etc.)**
Trying to eFile a document

**To Reproduce**
1. Log in as an external user
2. eFile a document larger than 2 MB
3. Note that the PDF viewer modal is blank
4. Switch document is one smaller than 2 MB
5. Note that the PDF displays in the PDF viewer modal

**Expected Behavior**
* A user can preview the eFiled document in the PDF viewer modal regardless of size

**Actual Behavior**
* PDFs larger than 2MB do not appear to be showing up in the PDF viewer modal on the "Review Your Filing" screen

**Screenshots**
Document larger than 2MB: [https://app.zenhub.com/files/152320868/ae3823c8-41f7-4ee2-b7fb-3c053021612f/download](https://app.zenhub.com/files/152320868/ae3823c8-41f7-4ee2-b7fb-3c053021612f/download)
![image.png](https://images.zenhubusercontent.com/5bbfb4d092b88e6ddb8b0e3d/50d08418-5629-4d83-b727-40be473c92d5)

Document under 2MB: [https://app.zenhub.com/files/152320868/eb089a7a-e49b-4bf7-a388-64ae108636f0/download](https://app.zenhub.com/files/152320868/eb089a7a-e49b-4bf7-a388-64ae108636f0/download)
![image.png](https://images.zenhubusercontent.com/5bbfb4d092b88e6ddb8b0e3d/9949034a-9a37-4633-a0c1-2d95f5ee57c9)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Cause of Bug, If Known**

**Process for Logging a Bug:**
* Complete the above information
* Add a severity tag (Critical, High Severity, Medium Severity or Low Severity). See below for priority definition. 

**Severity Definition:**
* Critical Defect
Blocks entire system's or module’s functionality
No workarounds available
Testing cannot proceed further without bug being fixed.

* High-severity Defect
Affects key functionality of an application
There's a workaround, but not obvious or easy
App behaves in a way that is strongly different from the one stated in the requirements

* Medium-severity Defect
A minor function does not behave in a way stated in the requirements.
Workaround is available and easy

* Low-severity Defect
Mostly related to an application’s UI
Doesn't need a workaround, because it doesn't impact functionality

**FOR ENGINEERING TEAM ONLY**
Bug Resolution Steps:
 - [x] Determine why the bug wasn't caught by a test. 
 - [x] Determine if an automated test needs to fixed, expanded or created. If unsure, bring in others to discuss.
 - [x] Determine if a manual test needs to be fixed, expanded or created. If unsure, bring in others to discuss.
 - [x] If needed, automated test is created.
 - [x] If needed, manual test is created.
 - [x] Reason for bug has been documented.
 - [x] Fix has been deployed to dev environment.
 - [x] Fix has been deployed to the Court's migration environment.
 - [x] Bug has been tested in Court's migration environment (UX or Engineering).

**Comment by kkoskelin** [comment]
Uncovered information relating to this bug here: https://bugs.chromium.org/p/chromium/issues/detail?id=69227
TL;DR: we cannot load base64 data URIs more than 2MB without killing the renderer.

Instead, we are using URL.createObjectURL() functionality -- which has a limit of 500MB.

PR: ustaxcourt/ef-cms-flexion#7790
